### DISCUSSION TITLE
Added user don't show up in the app's user table

### DISCUSSION THREAD

**Discussion by g-roll** [Budibase/budibase#3329] [Help] [answered]
Hey guys,

first of all: great app. Will definitely be a gamechanger 💪🏻

However, I would like to address something strange: I don't know if it's a bug or if I'm just missing something.  That's why I didn't open an issue in the first place.

If I add a user in admin-mode via the user management, he can access the apps (if I set this). However, his data does not appear in the BudibaseDB user table, of the respective app.

### **Problem:**

This is a problem because I want to apply conditional logic to display user specific content. Therefore it is not possible to use data bindings like {{ Current User.SOMEFILTER }}. Unfortunately, "vanilla" data bindings like {{ Current User.email }} don't work either. It will show everything to the user that is not listed in the user table.

**User management:**
![Screenshot 2021-11-11 at 09 41 28](https://user-images.githubusercontent.com/83816302/141269501-55f33862-f9ad-4f0b-8069-903b68a93291.png)

**App user table:**
![Screenshot 2021-11-11 at 09 40 25](https://user-images.githubusercontent.com/83816302/141269509-be129aee-1029-44ba-9b43-b842a88c9c5e.png)

### **Solution:**
I found out in Fauxton that in the app specific BudibaseDB, the ro_ta_users entry with the automatic userid, of the new user, is missing the entry "tableId": "ta_users" in the document.  If I enter this entry manually, the user appears in the user table as desired.

Mysteriously, everything works as expected. Each of the two users is only shown what I have assigned to him. Not as before, when this simple entry is missing.

**Fauxton (Blue: Missing entry - Yellow: UserId of the new user):**
![Screenshot 2021-11-11 at 09 39 00](https://user-images.githubusercontent.com/83816302/141269513-89dba658-22ec-4863-b740-864e7e9d9874.png)

**Updated app user table:**
![Screenshot 2021-11-11 at 09 50 41](https://user-images.githubusercontent.com/83816302/141269519-7c4416cc-992e-4017-9a69-ebaa652fd5c4.png)

### **Conclusion:**

In the business workflow I will have to use the API of CouchDB (not "BudibaseDB") that this entry is added automatically with the user registration process.

Is this a bug? Or did I miss something in the settings?

I have searched the documentation for this problem. Without success.
Likewise the issues and discussions/help on github. 

Greetings

**Comment by mike12345567** [comment, marked as answer]
Hi @g-roll, the version we will be releasing soon addresses this issue, it will make sure any user that is created that has access to an app will appear in the user metadata table within the app.

You can see the PR for this feature here: https://github.com/Budibase/budibase/pull/3251

**Reply by g-roll** [reply]
Thanks for the fast response.

Sorry, haven't looked in the dev branch so far 🙄

Greetings ✌🏻
